.flake8

```diff
@@ -1,6 +0,0 @@
-[flake8]
-count = True
-statistics = True
```

Comment:
what's the new way to get count + stats?

Reply:
Count according to flake8 [here](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-count) refers to printing the number of errors. Ruff, by default prints that

Reply:
For statistics, seems like the same thing. This is how a ruff sample output looks like:
```
poetry run ruff check guardrails/ tests/
guardrails/demo.py:1:8: F401 [*] `json` imported but unused
guardrails/demo.py:3:8: F401 [*] `openai` imported but unused
guardrails/demo.py:10:5: F841 Local variable `a` is assigned to but never used
guardrails/demo.py:11:5: F841 Local variable `b` is assigned to but never used
Found 4 errors.
[*] 2 fixable with the `--fix` option (2 hidden fixes can be enabled with the `--unsafe-fixes` option).
make: *** [lint] Error 1
```

Has both the total count and description for each error is shown